themes/hugo-bulma-blocks-theme/assets/sass/style.sass

```diff
@@ -127,4 +127,17 @@ code
   border-radius: 10px
   color: #4d6370 !important
   select
-    max-width: 50vw
\ No newline at end of file
+    max-width: 50vw
+
+.hero.user-conference
+  .hero-text
```

Comment:
I did consider that, but the issue with that is that there things in so many places in the hero right now; the main text to the top left, the icon to the top right, the tabs to the bottom left. So whichever direction you do the gradient from you'll end up with something on the image background.

It's also hard to move things around because of the "layout" of the image itself - if the main text where moved to the right then that would deviate from all the other heros, if the rest is moved to the left then most of the water (a main feature of the image) would be hidden by the gradient), and the tabs would be above the sidebar menu rather then the content, which IMO would seem weird.

Another option I've just tried is adding a slight black tint and blur to the hero background:
![image](https://github.com/user-attachments/assets/2f6712df-2370-4de5-9ee0-925eb070af4f)
Would that maybe be an option?

Reply:
Yes, I think adding a slight black tint is a good option. Perhaps at 50% so we don't have to make it blurry. After, we can remove the text shadow. 

What do you think about that? This is an example of 50% with the dark blue colour #002033:

![image](https://github.com/user-attachments/assets/a384f3ae-7069-4527-a175-7a77b40748a9)

Reply:
That looks good to me, I kept the text shadows however (altough more muted than before and also same color as the tint) to make the text stand out a bit more:

![image](https://github.com/user-attachments/assets/44eebaaa-7f0e-4b0b-8550-5e2a71919d7d)

Reply:
Looks good, thank you so much.